website/content/api-docs/secret/transform.mdx

```diff
@@ -1376,6 +1376,107 @@ $ curl \
 }
 ```
 
+## Lookup Token
+
+This endpoint returns the token given a plaintext and optionally an
+expiration or range of expirations.  This operation is only supported
```

Comment:
'This' as in 'this operation' is an antecedent so I'm not sure what 'this' is is referring to. If it's lookup token, perhaps it might be better to say Lookup Token is only supported...